src/devcenter/HISTORY.rst

```diff
@@ -2,10 +2,17 @@
 
 Release History
 ===============
+1.0.0
+++++++
+* Update control plane and data plane APIs to v2023-04-01
+* Azure Deployment Environments GA release
```

Comment:
Please add the renamed commands in release history. Because it's a breaking change.